src/QsCompiler/LlvmBindings/BitcodeModule.cs

```diff
@@ -120,19 +118,60 @@ private BitcodeModule(LLVMModuleRef handle)
             }
         }
 
-        /// <summary>Gets the <see cref="DebugInfoBuilder"/> used to create debug information for this module</summary>
-        /// <remarks>The builder returned from this property is lazy constructed on first access so doesn't consume resources unless used.</remarks>
+        /// <summary>Creates and returns a new <see cref="DebugInfoBuilder"/> used to create debug information for this module</summary>
+        public DebugInfoBuilder CreateDIBuilder()
+        {
+            this.ThrowIfDisposed();
+            DebugInfoBuilder dIBuilder = new DebugInfoBuilder(this);
+            this.DIBuilders.Add(dIBuilder);
+            return dIBuilder;
+        }
+
+        /// <summary>Gets the list of <see cref="DebugInfoBuilder"/>s used to create debug information for this module</summary>
+        /// <remarks>Each DebugInfoBuilder can own one paired compile unit.</remarks>
+        public List<DebugInfoBuilder> DIBuilders { get; }
+
+        /// <summary>
+        /// If there is exactly one <see cref="DebugInfoBuilder"/> used to create debug info for this module,
+        /// we return it through a get. This field is retained for backward compatibility.
```

Comment:
It's because DIBuilder was already exposed as public. Bettina and I had talked about everything that was exposed as public needing to remain exposed. @bettinaheim any thoughts?